### PR TITLE
add profile feature and get profile pic from gravatar

### PR DESCRIPTION
**Pull request by danielraad**
- added a profile modal that opens by tapping on a profile icon on the top left
- using a cordova native storage plugin to store the profile using key-value pairs
- using gravatar to get user's profile pictures

### CODE REVIEW

File: src/pages/home/home.scss [hunk 1]
```diff
@@ -3,4 +3,10 @@ page-home {
         height: 100%;
         width: 100%;
     }
+    #profile-icon {
+        margin: 10px;
+        height: 40px;
+        width: 40px;
+        border-radius: 50%;
```
Comment (arodr967) [inline src/pages/home/home.scss]: The profile icon should not be a circle, it should be a rounded-edged square. Maybe just adding 4px of border-radius will do the trick?

File: src/pages/home/home.ts [hunk 2]
```diff
@@ -46,6 +52,33 @@ export class HomePage {
       .catch(error => alert(`An error has occured:\n ${error}`));
   }
 
+  presentProfileModal() {
+    const profileData = this.getProfileData("email", "firstname", "lastname");
+    console.log("firstname " + profileData.firstname);
+    const profileModal = this.modalCtrl.create(ProfilePage, {profileData});
```
Comment (arodr967) [inline src/pages/home/home.ts]: I'm not sure how this is going to look but the profile page should not be a modal, it should just be it's own page

File: src/pages/profile/profile.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { IonicPage, ViewController, NavParams } from 'ionic-angular';
+
+interface Profile {
+  firstname: string;
+  lastname: string;
```
Comment (arodr967) [inline src/pages/profile/profile.ts]: **Best Practice:** Use camel-casing

**Review by arodr967 — COMMENTED** [review]
Good job!!

File: src/pages/home/home.ts (hunk 2 above)
Comment (danielraad) [reply src/pages/home/home.ts]: Addressed all comments but this one, when trying to use a normal page there is a weird lag when switching pages. I will look into it further tomorrow.

**Review by arodr967 — APPROVED** [review]

**Pull request merged**